### ISSUE TITLE
Gamle HL7 Norge profiler har overlapp med no-basis

### ISSUE THREAD

**Issue by thomiz** [HL7Norway/basisprofiler-r4#114]
Disse bør utfases/fjernes:
[Gammel HL7 Norge IG](https://www.hl7.no/fhir/profiles/0.3/Chapter.1.Welcome/index.html)

hl7.no bør være publiseringsplattform for nye kvalitetssikrede profiler fra HL7 Norge.

**Comment by rockphotog** [comment]
Ja, det er litt til tilbakevennende temaet om å rydde litt på HL7 Norge sine hjemmesider. Ta opp på TSK ved neste/beste anledning, @oaassv ?

**Comment by rockphotog** [comment]
Purring @oaassv @lisaFHI :-D

**Comment by oaassv** [comment]
Line har meldt gode nyheter fra Phoneix - nå blir det mulig å oppdatere hjemmesider i et felles regime og med budsjettstøtte fra HL7 International.

**Comment by rockphotog** [comment]
Hurra!
